src/dfa.rs

```diff
@@ -301,6 +301,8 @@ impl<'a> Iterator for InstPtrs<'a> {
     }
 }
 
+impl<'a> FusedIterator for InstPtrs<'a> {}
+
```

Comment:
I don't think `InstPtrs` is exposed anywhere, so I don't think we need to impl `FusedIterator` for it?